README.md

```diff
@@ -1,79 +1,89 @@
 
-The v2 branch is taking all the best lessons learned from the original HoloToolkit and refactoring / restructuring it to:
+* Enable rapid prototyping by providing the basic building blocks for MR app development. 
 
-* Support a wider audience, allowing solutions to be built that will run on multiple VR / AR / XR platforms, starting with Windows Mixed Reality and OpenVR.
+* Showcase best practices in MR with UI controls and interactions that matches the WMR and HoloLens Shell. 
 
-* Provide an easier to use SDK, to enable rapid prototyping and ease adoption for new users (or users of previous frameworks)
+* Support a wide audience, allowing solutions to be built that will run on multiple VR / AR / XR platforms such as Mixed Reality,  Steam/Open VR.
 
 * Ensure an extensive framework for advanced integrators, with the ability to swap out core components with their own should they wish to, or simply extend the framework to add new capabilities.
 
-> Learn more about [Windows Mixed Reality](https://www.microsoft.com/en-gb/windows/windows-mixed-reality) here.
 
-> Learn more about the architecture behind [Windows Mixed Reality - V2](MRTK-vNext.md) here.
+ # Required Software
+| [![Windows 10 Creators Update](External/ReadMeImages/MRTK170802_Short_17.png)](https://www.microsoft.com/software-download/windows10) [Windows 10 FCU](https://www.microsoft.com/software-download/windows10)| [![Unity](External/ReadMeImages/MRTK170802_Short_18.png)](https://unity3d.com/get-unity/download/archive) [Unity 3D](https://unity3d.com/get-unity/download/archive)| [![Visual Studio 2017](External/ReadMeImages/MRTK170802_Short_19.png)](http://dev.windows.com/downloads) [Visual Studio 2017](http://dev.windows.com/downloads)| [![Simulator (optional)](External/ReadMeImages/MRTK170802_Short_20.png)](https://go.microsoft.com/fwlink/?linkid=852626) [Simulator (optional)](https://go.microsoft.com/fwlink/?linkid=852626)|
+| :--- | :--- | :--- | :--- |
+| To develop apps for mixed reality headsets, you need the Windows 10 Fall Creators Update | The Unity 3D engine provides support for building mixed reality projects in Windows 10 | Visual Studio is used for code editing, deploying and building UWP app packages | The Emulators allow you test your app without the device in a simulated environment |
 
-> Learn more about the approach behind the [Windows Mixed Reality - V2 SDK](MRTK-SDK.md) here.
 
-# Getting Started with MRTK 
+# Supported Platform 
 
-You can find out how to use MRTK to develop for Windows Mixed Reality on the MS Developer Site.
+The Mixed Reality Toolkit vNext will includes many APIs to accelerate the development of MR / XR / VR / AR projects for a range of supported devices, starting with
 
-Find this readme, other documentation articles and the MRTK api reference on our [MRTK Dev Portal on github.io](https://microsoft.github.io/MixedRealityToolkit-Unity/). 
+ - Microsoft HoloLens
```

Comment:
Can we list HoloLens and HoloLens 2 as separate lines?

Reply:
Done